### PR TITLE
Change hard-coded cloud-init username

### PR DESCRIPTION
**Pull request by avivtur**
## 📝 Description

When creating a VM from IT, we want the cloud-init username would be similar to what we had when we create a VM from template.

### Before:

https://github.com/kubevirt-ui/kubevirt-plugin/assets/67270715/711c255e-29c6-42cc-92e5-cc09df7af159

### After:

https://github.com/kubevirt-ui/kubevirt-plugin/assets/67270715/a5d58776-30c2-4020-a03d-85a800981af4

### CODE REVIEW

File: src/views/catalog/CreateFromInstanceTypes/utils/utils.ts [hunk 1]
```diff
@@ -103,7 +115,7 @@ export const generateVM = (
       },
       preference: {
         // inferFromVolume: `${virtualmachineName}-disk`,
```
Comment (metalice) [inline src/views/catalog/CreateFromInstanceTypes/utils/utils.ts:117]: ?

Comment (avivtur) [reply src/views/catalog/CreateFromInstanceTypes/utils/utils.ts:117]: removed in here https://github.com/kubevirt-ui/kubevirt-plugin/pull/1305

**Pull request merged**